### PR TITLE
Feature/set up

### PR DESCRIPTION
**Pull request by ixboy**
Hello, this is my Math_Magicians  project, this is the first milestone with the following requirements :

1.  Initialise React app.
 

2.  Add React Redux.

3.  Structure your application files using a "feature folder" approach and use the ducks pattern for your Redux files. Your files/folder structure could look like this:
    ...
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | configureStore.js
        | index.js

4. The building blocks of your app should be set as re-usable components.
5. Add React Router and set two <Route>s and <Link>s for the app's navigation:
6. Books - the default view
            Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)
            Should have a form for adding a book (no functionality yet)
        Categories
            Should display "Under construction" text only.
    Styling is not required at this point.

I have implemented all the requirements, please do review my work and tell me if I can proceed to the next one.

Thank You

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const BookList = () => (
+  <div className="d-flex flex-row justify-content-center border">
+    <div className="d-flex flex-column px-3">
+      <h3>Programming</h3>
+      <h1>React for Dummies</h1>
+      <span>Author&apos;s name</span>
```
Comment (Davidosky007) [inline src/components/BookList.js:8]: * I strongly suggest you make this component take in and render book data ( `Title` and `Author`) as props, this is good practice as props enable you to pass properties from one component to another 
* For example you can render the `BookList` component multiple times with different data [Checkout this post on how to use Props](https://itnext.io/what-is-props-and-how-to-use-it-in-react-da307f500da0)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @ixboy 👋, 

Good job so far! 💪 
* Proper use of git-flow ✔️ 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Kindly make your `Readme file ` descriptive ( should contain information about your project and how to properly install and use it ). you can make use of t[his readme template ](https://github.com/microverseinc/readme-template) and customize it to suit your project 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.js (hunk 1 above)
Comment (ixboy) [reply src/components/BookList.js:8]: Thank You So much for the review, but this is just the first milestone of the project and I used dummy data, I will literally have to change everything in my code anyways.

**Review by D4nielJ — APPROVED** [review]
## Hello @ixboy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**